go.mod

```diff
@@ -46,7 +47,7 @@ require (
 	github.com/julienschmidt/httprouter v1.2.0
 	github.com/karalabe/usb v0.0.0-20190919080040-51dc0efba356
 	github.com/kylelemons/godebug v1.1.0 // indirect
-	github.com/ledgerwatch/secp256k1 v1.0.0 // indirect
+	github.com/ledgerwatch/secp256k1 v1.0.0
```

Comment:
Why is this modified?

Reply:
Not positive, but it's autogenerated by `go mod tidy`